pkg/controller/datavolume-controller.go

```diff
@@ -84,9 +90,9 @@ func NewDataVolumeController(
 	// Add datavolume-controller types to the default Kubernetes Scheme so Events can be
 	// logged for datavolume-controller types.
 	cdischeme.AddToScheme(scheme.Scheme)
-	glog.V(Vdebug).Info("Creating event broadcaster")
+	glog.V(3).Info("Creating event broadcaster")
```

Comment:
Why not using the _common_ const here?

Reply:
Yeah, surprisingly this seems to be the most controversial thing :)  So personally I'm used to just using glog and providing the arguments explicitly (1-5), k8s, kubevirt etc.  Just thought it would eliminate an import and was straight forward.  I also thought it was trivial, but it seems that maybe that's not the case.